spec/presenters/taxon_presenter_spec.rb

```diff
@@ -2,15 +2,20 @@
 
 RSpec.describe TaxonPresenter do
   describe "#payload" do
-    it "generates a valid payload" do
-      presenter = TaxonPresenter.new(
+    let(:presenter) do
+      TaxonPresenter.new(
         title: "My Title",
         base_path: "/taxons/my-taxon"
       )
+    end
+    subject { presenter.payload }
```

Comment:
I can see why the second version is more readable in isolation. My thinking was that in this case, we already have a `describe "#payload"` block, so `subject` is in that context (i.e. we are testing the payload).

I think it comes down to how we read the spec, though - either from top/bottom or looking at the expectations first. 

That said, I'm happy to drop `subject` in favour of `payload` if it's not helping.